wwwapp/migrations/0059_migrate_away_from_python_allaccess.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-03-07 06:29
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.conf import settings
+
+# After the migration is applied in production just comment out code depending on the allaccess library
+def forwards_func(apps, schema_editor):
+    AccountAccess = apps.get_model('allaccess', 'AccountAccess')
+    Provider = apps.get_model('allaccess', 'Provider')
+    UserSocialAuth = apps.get_model('social_django', 'UserSocialAuth')
+
+    for access in AccountAccess.objects.all():
+        if access.provider.name == 'facebook':
+            UserSocialAuth(provider='facebook', uid=access.identifier, user=access.user, extra_data=access.access_token).save()
+            access.delete()
```

Comment:
I'm not sure, should we be removing this automatically since we'll be tearing down the database for the old app anyway? If something goes horribly wrong while copying, there is no way to go back.
(yes, I know there should be a backup anyway)

Reply:
Agreed, migrations should not mix modify and delete steps.